Remove mention of tunnel (#1225)
saleor/saleor-docs: docs/developer/extending/apps/developing-with-tunnels.mdx

4. The tunnel will provide a publicly available URL of your application.
5. Now you can install your application in Saleor Cloud and receive webhooks 🎉.

## Example tunnels

> Saleor will work with any tunneling solution, andt they all follow a similar set-up process.

Reviewer: ```suggestion
Saleor will work with any tunneling solution and follow a similar set-up process.
```

> Here is the list of tunnels verified by us:

Reviewer: ```suggestion
Here are a few funnels we suggest:
```
